src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests.cs

```diff
@@ -5495,11 +5495,17 @@ class Goo<T>
 }
 ";
 
-            var compilation = CreateCompilation(source);
+            var compilation = CreateCompilation(source, parseOptions: TestOptions.Regular9);
             compilation.VerifyDiagnostics(
                 // (2,2): error CS0404: Cannot apply attribute class 'Goo<T>' because it is generic
```

Comment:
We should give a LangVersion diagnostic here which references the "generic attributes" feature.